jbpm-db-scripts/src/main/resources/db/ddl-scripts/sybase/sybase-jbpm-drop-schema.sql

```diff
@@ -1,92 +1,92 @@
-alter table Attachment drop constraint FK1C93543D937BFB5
+alter table Attachment drop constraint FKd5xpm81gxg8n40167lbu5rbfb
     go
-alter table Attachment drop constraint FK1C9354333CA892A
+alter table Attachment drop constraint FKjj9psk52ifamilliyo16epwpc
```

Comment:
probably all these changes are because of the reformat change. I think we should keep them as it is